FinalProject/gs-rest-service-master/complete/src/main/java/connector/DbConnector.java

```diff
@@ -0,0 +1,43 @@
+package connector;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DbConnector {
+	private final static String host = "localhost";
+	private final static String port = "3306";
+	private final static String name = "shoppingCart";
+	private final static String user = "root";
+	private final static String pass = "root";
+	private Connection connection = null;
+	private static DbConnector instance = null;
+	
+	private DbConnector() {
+		try {
+			Class.forName("com.mysql.jdbc.Connection");
+		} catch (ClassNotFoundException e) {
+			e.printStackTrace();

```

Comment:
Minor for this project: Don't forget printStackTrace() is for debug. To tell the user that something happend throw exception with a message.